src/core/shared/configurations/shared.ts

```diff
@@ -1,6 +1,8 @@
 export const SHARED_CONFIG = {
     // Default Respawn Time for Player
     RESPAWN_TIME: 25000,
+    // Default Respawn Key
+    RESPAWN_KEY: 88, // X
```

Comment:
This key should be configured in Death Plugin since it will be meaningless if the plugin is removed.

Reply:
[b6af687](https://github.com/Stuyk/altv-athena/pull/334/commits/b6af687a6576b49de3b419ea034a45599de7cdec)